### ISSUE TITLE
[BUU] Second line of Producer, Category and Tax category only visible half on Safari (iOS)

### ISSUE THREAD

**Issue by drummer83** [openfoodfoundation/openfoodnetwork#12828]
## Description

This is about the /products page for admin_style_v3.
On iOS Sonoma with Safari 17.3 (Browserstack) the content of the columns Producer, Category and Tax category gets wrapped to a second line, but that second line is not fully visible.

## Expected Behavior

Content should be truncated with ellipsis (...) or the second line has to be fully visible.

## Actual Behaviour

See screenshot.

## Steps to Reproduce

1. Go to /products.
2. Make sure you have some products with long (multiple words) producer names, categories or tax categories.
3.
4.

## Animated Gif/Screenshot

![image](https://github.com/user-attachments/assets/5aac7305-3412-42fd-a287-e9855cf84ec2)

## Workaround

None.

## Severity
bug-s4: it's annoying, but you can use it

## Your Environment

* Version used:
* Browser name and version:
* Operating System and version (desktop or mobile):

**Comment by RachL** [comment]
@drummer83 iOS as in mobile version? The screenshot you've shared suggest a desktop version - just checking

**Comment by drummer83** [comment]
@RachL No, desktop. Is it MacOS then? I'm a total stranger in the apple world. 😅

**Comment by RachL** [comment]
@drummer83 to be honest I did a quick research before commenting as I wasn't sure as well :joy: but yes I think we need to use "MacOS" in this case https://en.wikipedia.org/wiki/IOS
